### ISSUE TITLE
BlueKeep - Remote Code Execution (RDP Vulnerability) - CVE-2019-0708

### ISSUE THREAD

**Issue by lolminerxmrig** [projectdiscovery/nuclei-templates#9740]
Hi All
plz Add CVE-2019-0708 , Remote Code Execution (RDP Vulnerability).
thanks.

**Comment by princechaddha** [comment]
@lolminerxmrig, it would not be very straightforward to write a template for BlueKeep. We will try to write one using the JavaScript protocol and update here

**Comment by mastercho** [comment]
Yeah what happend with RDP module, plans were to make RDP module to test weak pass too
